torch/csrc/jit/python/python_sugared_value.cpp

```diff
@@ -785,6 +786,56 @@ std::shared_ptr<SugaredValue> createSimpleEnumValue(
   return toSimple(m.graph()->insertConstant(enum_ivalue, loc));
 }
 
+std::shared_ptr<SugaredValue> PythonSliceClass::call(
+    const SourceRange& loc,
+    Function& caller,
+    at::ArrayRef<NamedValue> inputs,
+    at::ArrayRef<NamedValue> attributes,
+    size_t /*n_binders*/) {
+  if (!attributes.empty()) {
+    throw ErrorReport(loc) << "Slice does not accept any keyword arguments";
+  }
+
+  static const int64_t default_start = 0;
+  static const int64_t default_stop = LLONG_MAX;
+  static const int64_t default_step = 1;
+  Graph& graph = *(caller.graph());
```

Comment:
Any suggestion on how to avoid it? Adding an assert here to ensure `graph` is not null? I though it is an invariant that `graph` is always not null when emitting IR. Otherwise, most of our code would segfault.